messages/de/AdminModule.base.php

```diff
@@ -25,7 +25,7 @@
   'Design' => 'Design',
   'Files' => 'Dateien',
   'General' => 'Allgemein',
-  'Groups' => 'Gruppen und Vereine',
+  'Groups' => 'Gruppen',
```

Comment:
can't this and the other two files with "wrong" changes be removed, as nothing else was changed?

Reply:
I don't know. Do you want me to remove it?

Reply:
You know what you have changed. If there are no changes anymore, we  do not need the files, this would mean: yes.